src/sagemaker/model_monitor/async_inference_config.py

```diff
@@ -0,0 +1,78 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Any suggestions for the folder to put this file and `serverless_inference_config.py`?